doc/nrf/known_issues.rst

```diff
@@ -321,6 +321,19 @@ NCSDK-9441: Fmfu SMP server sample is unstable with the newest J-Link version
 
   **Workaround:** Downgrade the debugger chip to the firmware released with J-Link 6.88a or use another way of transferring serial data to the chip.
 
+.. rst-class:: v2-1-0
+
+LTE Sensor Gateway fails to build
+  The :ref:`lte_sensor_gateway` sample fails to build when the board version is not included with the build target.
```

Comment:
Should be:

The :ref:`lte_sensor_gateway` sample system faults when not built board version. This is due to an issue with the low-power UART and HCI drivers.